crates/ol/chain-types/src/block.rs

```diff
@@ -134,28 +138,33 @@ pub struct OLBlockBody {
     tx_segment: OLTxSegment,
 
     /// Updates from L1.
-    l1_update: L1Update,
+    l1_update: Option<L1Update>,
```

Comment:
This means that not all OL blocks will have `L1Update`s?
Then, maybe update the docstring please?

Reply:
Correct.